code/modules/reagents/reagents/toxin.dm

```diff
@@ -506,3 +507,90 @@
 
 /datum/reagent/toxin/xeno_growthtoxin/overdose_crit_process(mob/living/L, metabolism)
 	L.Losebreath(2)
+
+
+/datum/reagent/toxin/xeno_hemodile //receives 50% increased stamina damage
+	name = "Hemodile"
+	description = "A stamina draining toxin. Causes increasing stamina loss the longer it is present."
+	reagent_state = LIQUID
+	color = "#94CFD8" // rgb: 148, 207, 216
+	custom_metabolism = 0.5
+	overdose_threshold = 10000
+	scannable = TRUE
+	toxpwr = 0
+
+/datum/reagent/toxin/xeno_hemodile/on_mob_add(mob/living/L, metabolism, affecting)
+	to_chat(L, "<span class='warning'>You have trouble catching your breath.</span>")
```

Comment:
On mob add?
This should probably be change to more "you suddenly feel your lungs contract" to indicate it was just injected or moved to a prob() in metabolize